community/libreoffice/APKBUILD

```diff
@@ -30,16 +30,44 @@ case $pkgver in
 	*.*.*) _v=$pkgver;;
 esac
 
-# grep LIBXMLSEC_TARBALL download.lst
+_ext_source="$_addsrcurl/134d8262145fc793c6af494dcace3e71-liberation-fonts-ttf-1.07.4.tar.gz          
+	$_addsrcurl/ba2930200c9f019c2d93a8c88c651a0f-flow-engine-0.9.4.zip
+	$_addsrcurl/17410483b5b5f267aa18b7e00b65e6e0-hsqldb_1_8_0.zip                            
+	$_addsrcurl/beeca87be45ec87d241ddd0e1bad80c1-bsh-2.0b6-src.zip                               
+	$_addsrcurl/c3c1a8ba7452950636e871d25020ce0d-pt-serif-font-1.0000W.tar.gz
+	$_addsrcurl/3404ab6b1792ae5f16bbd603bd1e1d03-libformula-1.1.7.zip                        
+	$_addsrcurl/c74b7223abe75949b4af367942d96c7a-crosextrafonts-carlito-20130920.tar.gz
+	$_addsrcurl/35c94d2df8893241173de1d16b6034c0-swingExSrc.zip                              
+	$_addsrcurl/ce12af00283eb90d9281956524250d6e-xmlsec1-1.2.20.tar.gz
+	$_addsrcurl/35efabc239af896dfb79be7ebdd6e6b9-gentiumbasic-fonts-1.10.zip                 
+	$_addsrcurl/commons-logging-1.2-src.tar.gz
+	$_addsrcurl/368f114c078f94214a308a74c7e991bc-crosextrafonts-20130214.tar.gz              
+	$_addsrcurl/d8b5214d35bcd2bfcb2cffa7795b351d-dejavu-fonts-ttf-2.35.zip
+	$_addsrcurl/39bb3fcea1514f1369fcfc87542390fd-sacjava-1.3.zip                             
+	$_addsrcurl/d8bd5eed178db6e2b18eeed243f85aa8-flute-1.1.6.zip
+	$_addsrcurl/3bdf40c0d199af31923e900d082ca2dd-libfonts-1.1.6.zip                          
+	$_addsrcurl/db60e4fde8dd6d6807523deb71ee34dc-liblayout-0.2.10.zip
+	$_addsrcurl/5c781723a0d9ed6188960defba8e91cf-liberation-fonts-ttf-2.00.1.tar.gz          
+	$_addsrcurl/e7a384790b13c29113e22e596ade9687-LinLibertineG-20120116.zip
+	$_addsrcurl/798b2ffdc8bcfe7bca2cf92b62caf685-rhino1_5R5.zip                              
+	$_addsrcurl/edc4d741888bc0d38e32dbaa17149596-source-sans-pro-2.010R-ro-1.065R-it.tar.gz
+	$_addsrcurl/7a15edea7d415ac5150ea403e27401fd-open-sans-font-ttf-1.10.tar.gz              
+	$_addsrcurl/eeb2c7ddf0d302fba4bfc6e97eac9624-libbase-1.1.6.zip
+	$_addsrcurl/8ce2fcd72becf06c41f7201d15373ed9-librepository-1.1.6.zip                     
+	$_addsrcurl/f94d9870737518e3b597f9265f4e9803-libserializer-1.1.6.zip
+	$_addsrcurl/907d6e99f241876695c19ff3db0b8923-source-code-pro-2.030R-ro-1.050R-it.tar.gz
+	$_addsrcurl/97b2d4dba862397f446b217e2b623e71-libloader-1.1.6.zip                         
+	$_addsrcurl/ace6ab49184e329db254e454a010f56d-libxml-1.1.7.zip
+	http://dev-www.libreoffice.org/extern/185d60944ea767075d27247c3162b3bc-unowinreg.dll"  
```

Comment:
These are the tarballs automatically downloaded by configure if --disable-fetch-external is not defined. configure then also downloads libreoffice-translations-$pkgver.tar.xz, but chokes on this tarball during extraction (!).
This is why the rest of the tarballs are downloaded and processed separately from libreoffice-translations.

Configure/make will demand that they are downloaded though (for example, it will stop if the dll is not present)